freedom-e-sdk/bsp/env/freedom-e300-hifive1/flash.lds

```diff
@@ -155,7 +155,7 @@ SECTIONS
   .stack ORIGIN(ram) + LENGTH(ram) - __stack_size :
   {
     PROVIDE( _heap_end = . );
-    . = __stack_size;
+    . += __stack_size;
```

Comment:
That might be why the stack_size setting never seemed to work.

Reply:
Yeah, I think so too.

Reply:
To clarify, this fixes a bug in the original linker script. GNU ld accepts this, but lld rightly throws an error.